docs/List.md

```diff
@@ -20,7 +20,7 @@ Elements can have fixed or varying heights.
 | rowHeight | Number or Function | ✓ | Either a fixed row height (number) or a function that returns the height of a row given its index: `({ index: number }): number` |
 | rowRenderer | Function | ✓ | Responsible for rendering a row. Signature should look like `({ index: number, key: string, style: Object, isScrolling: boolean }): React.PropTypes.node` and the returned element must handle index, key and style.|
 | scrollToAlignment | String |  | Controls the alignment scrolled-to-rows. The default ("_auto_") scrolls the least amount possible to ensure that the specified row is fully visible. Use "_start_" to always align rows to the top of the list and "_end_" to align them bottom. Use "_center_" to align them in the middle of container. |
-| scrollToIndex | Number |  | Row index to ensure visible (by forcefully scrolling if necessary) |
+| scrollToRow | Number |  | Row index to ensure visible (by forcefully scrolling if necessary) |
```

Comment:
[`scrollToIndex` is the correct prop name](https://github.com/bvaughn/react-virtualized/blob/c0e6e687434998ae6d6c7f8cd4ecbc9f11309f7b/source/List/List.js#L74-L75) for `List`. (The underlying `Grid` uses a prop named `scrollToRow` but that's because it also has coluns to account for.)

[`scrollToRow` is a method name](https://github.com/bvaughn/react-virtualized/blob/c0e6e687434998ae6d6c7f8cd4ecbc9f11309f7b/source/List/List.js#L125-L131) for `List` which I understand is a bit arbitrary but it _accepts a `List` parameter_ so there's some meaning there.